docs/gateways/OnDemand.md

```diff
@@ -193,10 +193,77 @@ Now that we have our environment correctly created, we can launch a Jupyter sess
 
-**_Notes:_** 
-* We have designed the RStudio app in Open OnDemand such that it employs versions of R that match the versions of R that are also available in the CURC module stack. This is done to facilitate moving between using RStudio for interactive work, and running larger R workflows as batch jobs on Alpine or Blanca. Due to system constraints, packages you install in a given version of R in RStudio will not be available if you load the equivalent version of the R module, and vice versa.  You will need to (re-)install the packages you need when using the equivalent module.
+**_Important Notes:_** 
+* We have designed the RStudio app in Open OnDemand such that it employs versions of R that match the versions of R that are also available in the CURC module stack. This is done to facilitate moving between using RStudio for interactive work, and running larger R workflows as batch jobs on Alpine or Blanca. Due to system constraints, packages you install in a given version of R in RStudio will not be available if you load the equivalent version of the R module, and vice versa. You will need to (re-)install the packages you need when using the equivalent module. This is due to the fact that RStudio is run from an Ubuntu [container](../Software/Containerizationon.html).
+
+###### Installing dependencies for RStudio (only available on Alpine)
+
+As previously mentioned, the RStudio application is run from an Ubuntu [container](../Software/Containerizationon.html). More specifically, the application uses an Ubuntu container paired with a [persistent overlay](https://apptainer.org/docs/user/main/persistent_overlays.html) that is unique to each user. For this reason, when installing a library via `install.packages`, you may receive an error because the container and overlay do not have a dependency required by the library. For example, let's try to install the library `XVector` using the Bioconductor package manager `BiocManager`, using the below commands in the R command prompt.
+```
+install.packages("BiocManager")
+library(BiocManager)
+BiocManager::install("XVector")
+```
+- Please note that if you are ever provided the prompt "Update all/some/none? [a/s/n]:",  always choose "n". You will not be able to update the items because RStudio needs to be launched using a read only container, which cannot be modified. However, choosing the wrong option should not harm anything.
+
+When the above lines are executed, we will eventually reach a state in the install where we receive the following error (caused by the `XVector` install).
```

Comment:
(caused by the `XVector` install) -> I think this could be written as "When the above lines are executed, we will eventually reach a state in the XVector install where we receive the following error."